addons/l10n_ar/models/l10n_latam_document_type.py

```diff
@@ -6,6 +6,8 @@ class L10nLatamDocumentType(models.Model):
 
     _inherit = 'l10n_latam.document.type'
 
+    name = fields.Char("Name", translate=True)
+    report_name = fields.Char("Report name", translate=True)
```

Comment:
Wouldn't it make sense to do this in the l10n_latam module? I suspect that this will be needed for every latam module no?

Reply:
I've done it in the Chilian and Equatorian localization already but i could do this in the l10n_latam and remove the field of those localisation in another PR ?

Reply:
Maybe you can just do separate commits in this pr?